pkg/chartutil/coalesce.go

```diff
@@ -47,10 +47,7 @@ func CoalesceValues(chrt *chart.Chart, vals map[string]interface{}) (Values, err
 	if valsCopy == nil {
 		valsCopy = make(map[string]interface{})
 	}
-	if _, err := coalesce(chrt, valsCopy); err != nil {
-		return valsCopy, err
-	}
-	return coalesceDeps(chrt, valsCopy)
+	return coalesce(chrt, valsCopy)
```

Comment:
You did not update the docs that explain why this was called twice (lines 38-39 above).

Reply:
Sorry, I overlooked the comment at lines 38 - 39. I think we can just remove that comment, or we can change the comment to "create a copy of vals and then pass it to coalesce, as it will mutate the passed values", because `coalesceDeps` was invoked in `colesce`, so that invoke was duplicated.

Reply:
I removed the comment in a followw-up PR because I wanted to get this merged in time for Helm 3.2 RC.1. Thanks again!